namesys/publisher.go

```diff
@@ -208,7 +194,7 @@ func PublishEntry(ctx context.Context, r routing.ValueStore, ipnskey string, rec
 	}
 
 	log.Debugf("Storing ipns entry at: %s", ipnskey)
-	// Store ipns entry at "/ipns/"+b58(h(pubkey))
+	// Store ipns entry at "/ipns/"+h(pubkey)
```

Comment:
wait, has this changed?

Reply:
I didn't change the relevant code, I just noticed that the comment was inaccurate.
[It seems like](https://github.com/ipfs/go-ipfs/blob/master/namesys/publisher.go#L372) the DHT key is `"/ipns/" + string(<multihash>)`
Note that in the IPNS path exposed to the user the hash is B58 encoded. The resolver [decodes it](https://github.com/ipfs/go-ipfs/blob/master/namesys/routing.go#L127) to get the DHT key.

Reply:
ok, cool. yeah, let's fix the comment if it's inaccurate.